modules/claims_api/config/schemas/2122.json

```diff
@@ -5,72 +5,6 @@
   "required": [
     "serviceOrganization"
   ],
-  "definitions": {
```

Comment:
more curious than anything, why move away from these definitions defined up here? im guessing rswag / the developer portal didnt like it this way or something?